README.md

```diff
@@ -15,16 +15,43 @@ This tool generate tree graph from ARF xml report from OpenSCAP scan.
   * [inquirer](https://pypi.org/project/inquirer/)
 
 ### Installation
+#### Fedora 30 and later
+```bash
+dnf install oval-graph
+# Install inquirer, if you want have nice cli futures. (optional)
+pip3 install inquirer
+```
+#### Install via [pip](https://pypi.org/project/oval-graph/)
+```bash
+pip3 install --user oval-graph
```

Comment:
Please describe here that scripts will be installed for the current user only, i.e. to `$HOME/.local/bin` - if users don't have that directory as part of system `PATH`, then they won't be able to use it.